packages/edit-site/src/components/page-patterns/dataviews-patterns.js

```diff
@@ -235,12 +253,33 @@ export default function DataviewsPatterns() {
 				render: ( { item } ) => (
 					<Title item={ item } categoryId={ categoryId } />
 				),
-				maxWidth: 400,
 				enableHiding: false,
 			},
-		],
-		[ view.type, categoryId ]
-	);
+		];
+		if ( type === PATTERN_TYPES.theme ) {
+			_fields.push( {
+				header: __( 'Sync Status' ),
+				id: 'sync-status',
+				render: ( { item } ) => {
+					// User patterns can have their sync statuses checked directly.
+					// Non-user patterns are all unsynced for the time being.
+					const [ , notSyncedStatus ] = SYNC_FILTERS;
+					return ! item.syncStatus
```

Comment:
What do you think of doing this instead:

```js
return ! item.syncStatus
    ? PATTERN_SYNC_TYPES.unsynced
    : SYNC_FILTERS.find( ( { value } ) => value === item.syncStatus )?.label;
```

From a reader point of view, I find it much clearer. It's also more robust because this won't break if anyone reorders the `SYNC_FILTERS` array, or adds a new item, etc.

Reply:
`PATTERN_SYNC_TYPES` only contains the values and not the labels. Regarding the deconstruct I think it's simple and not worth it to have an extra loop there. Do you have strong opinions on using an extra `array.find`?

Reply:
My stronger concern is extracting the label from the deconstruct. This bit is fragile (others can add to that array, reorder it, etc.):

> const [ , notSyncedStatus ] = SYNC_FILTERS;

I'd be fine with almost any approach that doesn't use it.